src/main/java/org/opentripplanner/routing/algorithm/raptor/transit/constrainedtransfer/ConstrainedBoardingSearch.java

```diff
@@ -33,7 +32,7 @@
     private static final ConstrainedBoardingSearchStrategy REVERSE_STRATEGY = new ConstrainedBoardingSearchReverse();
 
     /** Handle forward and reverse specific tasks */
-    private final ConstrainedBoardingSearchStrategy translator;
+    private final ConstrainedBoardingSearchStrategy fwdRvsStrategy;
```

Comment:
I'm not really happy with this variable name, but I'll let it slip, if there are no other changes requested.

Reply:
I can change that.